core/src/main/java/com/datastrato/gravitino/Config.java

```diff
@@ -1,11 +1,12 @@
 /*
- * Copyright 2023 Datastrato Pvt Ltd.
+ * Copyright 2024 Datastrato Pvt Ltd.
  * This software is licensed under the Apache License version 2.
  */
 package com.datastrato.gravitino;
```

Comment:
move `Config` from `common` module to `core` module, @jerryshao WDYT?

Reply:
If it's ok,  I will move `ConfigBuilder` and `ConfigEntry` and `ConfigConstants` too

Reply:
done